overlays/holo-nixpkgs/holo-cloudflare-wrangler/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cloudflare-wrangler";
```

`wrangler` is the canonical upstream name of this package, rather than `cloudflare-wrangler`.

```suggestion
  pname = "wrangler";
```

---

done